### DISCUSSION TITLE
Brevo Action

### DISCUSSION THREAD

**Discussion by seehat** [tobimori/kirby-dreamform#22] [Feature Requests]
- Subscribe users to a specific list on a form submission, and map form fields to Brevo fields
   https://developers.brevo.com/docs/synchronise-contact-lists

**Comment by tobimori** [comment]
I noticed Brevo offers not only Email but also SMS/WhatsApp campaigns. Is this interesting for you as well?

**Comment by seehat** [comment]
Not really. I mostly use EMail. Thx for asking. Great plugin!

**Comment by tobimori** [comment]
This is included in 1.5.0-rc.3.
